release/RELEASE.md

```diff
@@ -1,124 +1,142 @@
+   create a "split release" bundle and push it to Artifactory. It should notify
+   on `#ci-failures-daml` on Slack.
+4. The next step is to request a build from the [Canton] team (on
+   `#team-canton`) that relies on the RC snapshot. Once you have a [canton]
+   release, you can proceed.
+5. Go to the [assembly] repo, and follow the instructions there to make a release
+   using the [Canton] version that was just created. The `LATEST` file on the
+   [assembly] repo only contains on version; it is safe to overwrite it, and to
+   "go backwards" if needed.
+6. Once the `main` build of the [assembly] repo has finished, you should
+   proceed with testing. You should open up this document _in the branch of
+   the release you're making_, as testing instructions change over time.
+7. After testing, if everything went well, you should go on to turning the RC
+   into a stable release. If something went wrong, make appropriate changes to
+   the release branch and start over.
+
+## Stable 2.x
+
+Making a stable release follows the same steps as a snapshot RC, except that:
+
+- You should not be choosing an arbitrary commit, you should pick the latest RC
+  for the branch.
+- Go through the [checklist] before making the release. Note that
+  the checklist is not available publicly.
+- Instead of adding a line to [`LATEST`], remove the `-snapshot` part of the
+  version number for the existing RC.
+- Similarly, modifying the `LATEST` file on the assembly repo _should_ amount
+  to just removing the `-` parts of the version numbers for both Canton and
+  daml, but follow the instructions there.
+- You need a team lead to approve the PR on both the [daml] and [assembly] repos.
+
+## 1.x release candidate
+
+0. Make sure all the changes you want are in the release branch.
 1. Start _from latest `main`_ and run
    ```
    $ ./release.sh snapshot origin/release/1.18.x 1.18.3
-   5221a5d13e6d2e4b657abdd951103cd3030fae36 1.18.3-snapshot.20220310.8437.0.5221a5d1
    ```
-   The output line needs to be added to [`LATEST`] _on `main`_, either as a new
-   entry if there is no existing 1.18.3 line, or as a replacement for the existing
-   one if there is. Try to keep the [`LATEST`] file sorted by semver (highest
-   veersion on top) and containing only one entry for each version number. Make a
-   PR with just the changed [ `LATEST`] file _to the `main` branch_. Add the
-   `Standard-Change` label to that PR.
-
+   The output will be a line that starts with a commit sha, followed by a
+   snapshot release number. You need to take that line and add it to the [`LATEST`]
+   file. You should put that line in the file so as to preserve semver
+   ordering, and overwrite any existing snapshot with the same prefix.
+2. Make a PR against the `main` branch with just that one line added, touching
+   no other file. Add the `Standard-Change` label to that PR.
 1. Once the PR has built, check that it was considered a release build by our
```

Comment:
```suggestion
2. Once the PR has built, check that it was considered a release build by our
```
and so forth?

Reply:
Markdown doesn't care, it corrects when rendering. But fair point, I'll make a pass through.